lib/jsonapi/serializable.rb

```diff
@@ -1,3 +1,35 @@
-require 'jsonapi/serializable/error'
-require 'jsonapi/serializable/resource'
-require 'jsonapi/serializable/renderer'
+module JSONAPI
+  module Serializable
+    require 'jsonapi/serializable/error'
+    require 'jsonapi/serializable/resource'
+    require 'jsonapi/serializable/renderer'
+
+    # Error raised when there's no serializable class defined for resource.
+    class UndefinedSerializableClass < StandardError; end
+
+    # @api private
+    def self.resources_for(objects, options, inferrer)
```

Comment:
Any specific rationale for not defining those with `module_function`? I do not feel strongly about this but just wondering.

Reply:
AFAIK, `module_function` is meant to be used with modules that are mixed in classes, because it also creates private instance methods available to the instance methods of the class.

Since I don't think `JSONAPI::Serializable` will ever be included in any class, I didn't used `module_function`.

Reply:
Makes sense 👌

Reply:
@smaximov Thanks for sharing the explication of `module_function` :)